### PR TITLE
merge frontend implementation of everything minus checkout page and admin page

### PR DESCRIPTION
**Pull request by levkropp**
login page and register page have visible text and submit form buttons with functions ready for backend implementation
shopping cart works, can be passed to anything between the header and footer 
shopping cart page displays a table and checkout button

### CODE REVIEW

File: frontend/README.md
Comment (ksmarty) [inline frontend/README.md:1]: We don't need this file

File: frontend/firebase-debug.log
Comment (ksmarty) [inline frontend/firebase-debug.log:1]: We don't need this file

File: frontend/pages/certs.js
Comment (ksmarty) [inline frontend/pages/certs.js:1]: We don't need this, right?

**Review by ksmarty — CHANGES_REQUESTED** [review]

File: frontend/README.md
Comment (levkropp) [reply frontend/README.md:1]: yeah i guess

File: frontend/pages/certs.js
Comment (levkropp) [reply frontend/pages/certs.js:1]: no, this was a blank page i just copied when making new pages/Components but we don't need it in the repo

**Review by ksmarty — APPROVED** [review]

**Pull request merged**